src/service/bridge/components/WebviewWithBridge/index.tsx

```
@@ -0,0 +1,103 @@
+  Ref,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from "react";
+import WebView from "react-native-webview";
+import type { WebViewMessageEvent } from "react-native-webview/lib/WebViewTypes";
+
+interface WebViewWithBridgeProps<ReqMessage, ResMessage>
+  extends Omit<ComponentProps<typeof WebView>, "onMessage"> {
+  onBridgeMessage?: (reqMessage: ReqMessage) => ResMessage | void;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)